doc/source/installing.rst

```diff
@@ -431,7 +431,7 @@ source include:
 
 - ``git``
 - A C compiler such as ``gcc`` or ``clang``
-- ``Python 2.7``, ``Python 3.5``, or ``Python 3.6``
+- ``Python 2.7``, ``Python >= 3.5``
```

Comment:
I think since we're going through and taking out py2 testing, we should just recommend Python >= 3.5 here in the docs.

Reply:
Sure thing !